src/key.rs

```diff
@@ -148,6 +148,19 @@ impl SecretKey {
         }
     }
 
+    #[inline]
+    /// Negates one secret key.
+    pub fn negate_assign(
+        &mut self
+    ) {
+        unsafe {
+            ffi::secp256k1_ec_privkey_negate(
+                ffi::secp256k1_context_no_precomp,
+                self.as_mut_c_ptr()
+            );
+        }
```

Comment:
Please assert the return value with 1, this rn returns always 1(https://github.com/rust-bitcoin/rust-secp256k1/blob/master/secp256k1-sys/depend/secp256k1/include/secp256k1.h#L563) 
But in the future it might return 0 if the seckey is invalid, but our type system doesn't allow that to ever happen

Reply:
Thank you for your comment. I will update.

> But in the future it might return 0 if the seckey is invalid, but our type system doesn't allow that to ever happen

Excellent 🎉